com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/usecases/TestCobolStringWithContinuationLineConcatenatedCorrectly.java

```diff
@@ -13,32 +13,42 @@
  */
 package com.ca.lsp.cobol.usecases;
 
+import com.ca.lsp.cobol.usecases.engine.UseCaseEngine;
+import org.junit.Ignore;
 import org.junit.Test;
 
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptyMap;
+
 /**
- * This test checks that there are no errors thrown if a keyword split and concatenated by a
+ * This test checks that there are no errors thrown if a string literal split and concatenated by a
  * concatenation line.
  */
-public class TestCobolStringWithContinuationLineConcatenatedCorrectly extends PositiveUseCase {
+public class TestCobolStringWithContinuationLineConcatenatedCorrectly {
 
   private static final String TEXT =
-      "000100 IDENTIFICATION DIVISION.\r\n"
-          + "000200 PROGRAM-ID.   TEST1.\r\n"
-          + "000301 AUTHOR.       A PROGRAMMER.\r\n"
-          + "000400 PROCEDURE DIVISION.\r\n"
-          + "078070 000-000.\r\n"
-          + "078076     EVALUATE TRUE\r\n"
-          + "078081         WHEN PROGRAM1\r\n"
-          + "078089                 MOVE 'PRESS \"CLEAR\" OR \"ENTER\" TO RETURN TO THE M\r\n"
-          + "078089-                     'ENU WHEN FINISHED' TO ID1\r\n"
-          + "078091                 PERFORM PROGRAM2.";
-
-  public TestCobolStringWithContinuationLineConcatenatedCorrectly() {
-    super(TEXT);
-  }
+      "000100 IDENTIFICATION DIVISION.\n"
+          + "000200 PROGRAM-ID.   TEST1.\n"
+          + "000301 AUTHOR.       A PROGRAMMER.\n"
+          + "000000 DATA DIVISION.\n"
+          + "000000 WORKING-STORAGE SECTION.\n"
+          + "000000  01 {$*ID1}.\n"
+          + "000000  01 {$*ID2}.\n"
+          + "000400 PROCEDURE DIVISION.\n"
+          + "078070 {#*000-MAIN}.\n"
+          + "078076     EVALUATE TRUE\n"
+          + "078081         WHEN {$ID1}\n"
+          + "078089                 MOVE 'PRESS \"CLEAR\" OR \"ENTER\" TO RETURN TO THE M\n"
+          + "078089-                     'ENU WHEN FINISHED' TO {$ID2}\n"
+          + "078091                 PERFORM {#PROGRAM2}\n"
+          + "000000     END-EVALUATE.\n"
+          + "000000 {#*PROGRAM2}.\n"
+          + "000000  DISPLAY 'HELLO'.";
 
+  @Ignore("Continuation line cause a semantic error #414")
+  // TODO: #414
   @Test
   public void test() {
-    super.test();
+    UseCaseEngine.runTest(TEXT, emptyList(), emptyMap());
```

Comment:
List.of(), Map.of(),  immutable empty

Reply:
https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#emptyList--

emptyList() returns immutable list too

Reply:
Yes, but List.of() is much more safer, e.g. you can deserialize emptyList() on JDK 8 and previous, but not List.of.